### ISSUE TITLE
Supply action throwing undefined method length

### ISSUE THREAD

**Issue by MichaelPalmer1** [fastlane/fastlane#4010]
### New Issue Checklist

- [X] Updated fastlane to the latest version
- [X] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description
When I try to use supply as an action to update the metadata on Google Play, I get a ```undefined method `length' for nil:NilClass (NoMethodError)``` error. It looks like this is being caused by the recent update (a85a42f546cb607b261ac215f1e1c83a9e7e2b70) to supply that enables multiple apk uploads (specifically, it is happening on the second line below):

fastlane/lib/fastlane/actions/supply.rb:
```ruby
all_apk_paths = Actions.lane_context[SharedValues::GRADLE_ALL_APK_OUTPUT_PATHS]
if all_apk_paths.length > 1
    params[:apk_paths] ||= all_apk_paths
else
    params[:apk] ||= Actions.lane_context[SharedValues::GRADLE_APK_OUTPUT_PATH]
end
```

I'd recommend doing an additional check in the if statement to ensure that `all_apk_paths` is not `nil`. I'll be happy to make this fix upon confirmation of the issue.

##### Complete output when running fastlane, including the stack trace and command used

```
MacBook-Pro:app mpalmer$ fastlane android metadata
[16:39:02]: -----------------------------
[16:39:02]: --- Step: update_fastlane ---
[16:39:02]: -----------------------------
[16:39:02]: Looking for updates for cert, credentials_manager, deliver, fastlane, fastlane_core, frameit, gym, match, pem, pilot, produce, scan, screengrab, sigh, snapshot, spaceship, supply...
[16:39:06]: Nothing to update ✅
[16:39:06]: -------------------------------------------------
[16:39:06]: --- Step: Verifying required fastlane version ---
[16:39:06]: -------------------------------------------------
[16:39:06]: fastlane version valid
[16:39:08]: Driving the lane 'android metadata' 🚀
[16:39:08]: --------------------
[16:39:08]: --- Step: supply ---
[16:39:08]: --------------------
[16:39:09]: Variable Dump:
[16:39:09]: {:PLATFORM_NAME=>:android, :LANE_NAME=>"android metadata"}
[16:39:09]: undefined method `length' for nil:NilClass

+------+-------------------------------------+-------------+
|                     fastlane summary                     |
+------+-------------------------------------+-------------+
| Step | Action                              | Time (in s) |
+------+-------------------------------------+-------------+
| 1    | update_fastlane                     | 4           |
| 2    | Verifying required fastlane version | 0           |
| 3    | supply                              | 0           |
+------+-------------------------------------+-------------+

[16:39:09]: fastlane finished with errors
-------------------------------------------------------------------------------------------
😨  An error occured. Please enable crash reports using `fastlane enable_crash_reporting`.
👍  This makes resolving issues much easier and helps improve fastlane.
🔒  The reports will be stored securely on getsentry.com.
🙊  More information about privacy: https://github.com/fastlane/fastlane/releases/tag/1.33.3
-------------------------------------------------------------------------------------------
/Users/mpalmer/gems/gems/fastlane-1.69.0/lib/fastlane/actions/supply.rb:12:in `run': [!] undefined method `length' for nil:NilClass (NoMethodError)
	from /Users/mpalmer/gems/gems/fastlane-1.69.0/lib/fastlane/runner.rb:143:in `block (2 levels) in execute_action'
	from /Users/mpalmer/gems/gems/fastlane-1.69.0/lib/fastlane/actions/actions_helper.rb:35:in `execute_action'
	from /Users/mpalmer/gems/gems/fastlane-1.69.0/lib/fastlane/runner.rb:129:in `block in execute_action'
	from /Users/mpalmer/gems/gems/fastlane-1.69.0/lib/fastlane/runner.rb:128:in `chdir'
	from /Users/mpalmer/gems/gems/fastlane-1.69.0/lib/fastlane/runner.rb:128:in `execute_action'
	from /Users/mpalmer/gems/gems/fastlane-1.69.0/lib/fastlane/fast_file.rb:149:in `method_missing'
	from Fastfile:13:in `block (2 levels) in parsing_binding'
	from /Users/mpalmer/gems/gems/fastlane-1.69.0/lib/fastlane/lane.rb:36:in `call'
	from /Users/mpalmer/gems/gems/fastlane-1.69.0/lib/fastlane/lane.rb:36:in `call'
	from /Users/mpalmer/gems/gems/fastlane-1.69.0/lib/fastlane/runner.rb:52:in `block in execute'
	from /Users/mpalmer/gems/gems/fastlane-1.69.0/lib/fastlane/runner.rb:46:in `chdir'
	from /Users/mpalmer/gems/gems/fastlane-1.69.0/lib/fastlane/runner.rb:46:in `execute'
	from /Users/mpalmer/gems/gems/fastlane-1.69.0/lib/fastlane/lane_manager.rb:46:in `cruise_lane'
	from /Users/mpalmer/gems/gems/fastlane-1.69.0/lib/fastlane/command_line_handler.rb:30:in `handle'
	from /Users/mpalmer/gems/gems/fastlane-1.69.0/bin/fastlane:38:in `block (2 levels) in run'
	from /usr/local/lib/ruby/gems/2.2.0/gems/commander-4.3.5/lib/commander/command.rb:178:in `call'
	from /usr/local/lib/ruby/gems/2.2.0/gems/commander-4.3.5/lib/commander/command.rb:178:in `call'
	from /usr/local/lib/ruby/gems/2.2.0/gems/commander-4.3.5/lib/commander/command.rb:153:in `run'
	from /usr/local/lib/ruby/gems/2.2.0/gems/commander-4.3.5/lib/commander/runner.rb:428:in `run_active_command'
	from /Users/mpalmer/gems/gems/fastlane_core-0.41.0/lib/fastlane_core/ui/fastlane_runner.rb:23:in `run!'
	from /usr/local/lib/ruby/gems/2.2.0/gems/commander-4.3.5/lib/commander/delegates.rb:15:in `run!'
	from /Users/mpalmer/gems/gems/fastlane-1.69.0/bin/fastlane:156:in `run'
	from /Users/mpalmer/gems/gems/fastlane-1.69.0/bin/fastlane:162:in `<top (required)>'
	from /Users/mpalmer/gems/bin/fastlane:22:in `load'
	from /Users/mpalmer/gems/bin/fastlane:22:in `<main>'
```

### Configuration Files

Fastfile:
```ruby
# Check for fastlane updates
update_fastlane

# This is the minimum version number required.
fastlane_version '1.69.0'

#################
##   Android   ##
#################
platform :android do
    desc 'Update metadata on Google Play'
    lane :metadata do
        supply(
            track: 'alpha',
            skip_upload_apk: true,
            skip_upload_screenshots: true
        )
    end
end
```

AppFile:
```ruby
json_key_file "~/google-json-key.json"
package_name ENV["APP_ID"]
```

.env:
```bash
APP_ID='com.**********.************'
```
For privacy purposes, I have redacted the full app identifier.

### Environment

fastlane version:
- fastlane 1.69.0

Do you use bundler, rbenv or rvm?
- No

**Comment by barriosnahuel** [comment]
Hi! I've created #4009, so that's another occurrence of this issue. I will close that issue now because yours actually has more information about the issue.

**Comment by asfalcone** [comment]
Thanks for the report! Looking into a fix now.

**Comment by asfalcone** [comment]
A fix will go out in fastlane version 1.70.0 shortly! :rocket:
